_data/application.yml

```diff
@@ -1688,30 +1753,270 @@
 
     - label: "No"
 
-- title: Your plans
+- title: Plan checklist
```

Comment:
@brianleesfgov The two biggest changes in this version are:

- This multi-page "plan checklist" section
- The last page